### PR TITLE
Fix multiframe CRC calculation

### PR DESCRIPTION
**Pull request by vvg77ripe**

### CODE REVIEW

**Comment by vvg77ripe** [comment]
I think you may never used new libcanard to receipt multiframes, because usually small devices don't need them. Multiframe transmission, such as node.GetInfo response, works correctly.
Also i noticed that multiframe reception doesn't work with zero frame timestamps (because not_initialized check fails), but i don't know it is mandatory parameter or not.

**Comment by vvg77ripe** [comment]
Oops, I think i was little hurried and forgot to initialize CRC for next transfers. I hope this commit is more accurate.

File: canard.c [hunk 1]
```diff
@@ -372,7 +372,7 @@ int16_t canardHandleRxFrame(CanardInstance* ins, const CanardCANFrame* frame, ui
             return CANARD_ERROR_OUT_OF_MEMORY;
```
Comment (pavel-kirienko) [inline canard.c]: Could you please also fix this one and the one below on line 386 while you're at it?

**Comment by pavel-kirienko** [comment]
> Also i noticed that multiframe reception doesn't work with zero frame timestamps (because not_initialized check fails), but i don't know it is mandatory parameter or not.

Yep, the timestamps are mandatory.

We really should have our unit/regression tests organized. Any help in resolving https://github.com/UAVCAN/libcanard/issues/102 would be appreciated.

**Pull request merged**